src/Admin.java

```diff
@@ -0,0 +1,10 @@
+public class Admin extends User {
```

Comment:
Well, first of all take a look at packages structure
When you have a small application, might be it can be readable, but in case when a project starts to grow, it is necessary to build a good package structure so that everything is readable and understandable.
So think about how you can logically divide classes into packages